### PR TITLE
[no jira] Abstract out some theme props for DRYness

### PR DESCRIPTION
**Pull request by shaundon**
This replaces the hard-coded values with a mapping derived from the image @jamesf3rguson provided. It should mean for most uses we won't need to refer back to the image.

@matteo-hertel how come `colorBlue500` is in there - can I replace this with one of the themed colours?

### CODE REVIEW

**Comment by matthewdavidson** [comment]
This is blocked until #319

**Comment by matteo-hertel** [comment]
colorBlue is there because the nav for web will change so as part of the change I'll update that as well

File: packages/bpk-component-theme-toggle/src/theming.js [hunk 1]
```diff
@@ -19,14 +19,27 @@ import {
   colorBlue500,
 } from 'bpk-tokens/tokens/base.es6';
 
+/* eslint-disable no-unused-vars */
```
Comment (matthewdavidson) [inline packages/bpk-component-theme-toggle/src/theming.js]: Whats the unused var here?

Comment (shaundon) [reply packages/bpk-component-theme-toggle/src/theming.js]: The secondary ones are all unused - but I think it's good to have them in there for future usage. This is essentially acting as a mini version of `bpk-tokens` for theming variables (which we may not want inside `bpk-tokens` as they're just for usage here).

Comment (matthewdavidson) [reply packages/bpk-component-theme-toggle/src/theming.js]: Ah in that case it makes sense to wait until #315 because MY PR uses them. :)

Comment (shaundon) [reply packages/bpk-component-theme-toggle/src/theming.js]: Damn, you got me!

**Pull request merged**